### PR TITLE
Implemented final deployment

### PR DESCRIPTION
**Pull request by vangelif**
# In this project, I: 

- Used React documentation
- Used React components
- Used React props
- Used React Router
- Connected React and Redux
- Handled events in a React app
- Wrote integration tests
- Used styles in a React app
- Used React life cycle methods
- Applied React best practices and language style guides in code
- Used store, actions and reducers in React

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import './index.scss';
 import App from './App';
 import store from './redux/store';
```
Comment (Shisui6) [inline src/index.js:7]: Kindly add more comprehensive tests to your project. Currently, you only added snapshot tests which does not fulfil the project requirements shown below. Please add unit tests that test pure functions in your redux code as well as integration tests using functions from the react testing library🙏

image

Comment (Shisui6) [inline src/index.js:7]: Currently, there are many lines which disable linter checks in your project. Please go through your project and remove **all** lines that disable linter checks and try to solve the linter errors yourself. Learning to solve linter errors will help you learn best coding practices and make you a better developer☝️

Comment (Shisui6) [inline src/index.js:7]: According to the project requirements, the homepage should have a list of categories that could be filtered out by some parameter like country name. In your case, I suggest you implement a search bar or filter that allows users to filter the results on the homepage by continent.

image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @vangelif, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @vangelif , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**